docs/blog/september-contributions-showcase.md

```diff
@@ -0,0 +1,120 @@
+
+Paul ([@paulmillr](https://x.com/paulmillr/), the author of chokidar) and I ([@43081j](https://x.com/43081j)) have been working hard to get the next major version of it over the line for a long time now (the branch was made in 2021!).
+
+Recently, we finally released [4.0.0](https://github.com/paulmillr/chokidar/releases/tag/4.0.0)! :partying_face: this massively reduces the size and complexity.
+
+Those of you who remember the `is-number` fiasco on social media will now also be super happy that we (chokidar) were the largest consumer of it and have since dropped it entirely :tada:
+
+Big wins for the e18e community!
+
+Thanks to [@benjaminmccann](https://x.com/benjaminmccann) and talentlessguy for contributing _many_ v3 to v4 upgrades already!
+
+## fdir / tinyglobby
+
+`fdir` was in last month's showcase but has earned another mention thanks to the author ([thecodrr](https://x.com/thecodrr)) landing various community contributed PRs and features.
+
+This is a great example of where an author has joined the community and worked closely with it to further their library, and provide some much needed fixes and features.
+
+In version [6.4.0](https://github.com/thecodrr/fdir/releases/tag/v6.4.0), we saw two important changes:
+
+- Ability to exclude symlinks
+- Ability to bring your own glob library (e.g. [zeptomatch](https://github.com/fabiospampinato/zeptomatch))
+
+These two changes have helped the community migrate even more high level tools and frameworks across.
+
+Similarly, [tinyglobby](https://github.com/SuperchupuDev/tinyglobby) has shipped a few new versions. Particularly it is now able to determine if a string is a glob or not. This is super useful for migrating more projects to it.
+
+Thanks to pralkarz, [@benjaminmccann](https://x.com/benjaminmccann) and [@superchupudev](https://x.com/superchupu) for already migrating so many projects to these two libraries!
+
+## tinyexec migrations
+
+The community has been working wonders migrating countless projects from `execa` and other heavyweight libraries to the super lean [tinyexec](https://github.com/tinylibs/tinyexec/).
+
+For example, thanks to pralkarz, [vitest](https://github.com/vitest-dev/vitest) has already moved over!
+
+There are many more PRs currently in progress to do just the same :pray: Great work by the community here.
+
+# Contributions / Improvements
+
+## storybook
+
+The [storybook](https://storybook.js.org/) team have been working closely with the e18e community for some time now, dedicating huge amounts of time to improving the performance and install footprint for everyone.
+
+This month, they've kept up the pace! Just a few examples of what has landed:
+
+- Migrated from `chalk` to `picocolors`
+- Migrated from `fs-extra` to built in `fs` functions
+- Removed `handlebars` and simplified templating internally
+- Removed `prettier` from the core bundle (saved **4MB** on the install size)
```

Comment:
I double-checked the commits, this actually shaved off 10 MB because we're dual-publishing the bundle
```suggestion
- Removed `prettier` from the core bundle (saved **10MB** on the install size)
```